.github/workflows/unit-tests.yml

```diff
@@ -47,8 +47,9 @@ jobs:
     - name: Lint
       run: npm run lint
 
-    - name: Format
-      run: npm run format-check
+    # uncomment after reviewing rules
```

Comment:
Why do we need these changes?

Reply:
I had commented these out as part of prettier changes and forgot to uncomment as part of last checkin.  Unrelated to version bump

Reply:
Lets remove these changes for this PR then!

Reply:
sure. will do.